wikilabs/tocP/tiddlers/$__plugins_wikilabs_tocP_macros.tid

```diff
@@ -23,6 +25,13 @@ type: text/vnd.tiddlywiki
           </$list>
           <$transclude tiddler='$:/config/wikilabs/tocP/newChild'></$transclude>
           <$macrocall $name="tocP-body" tag=<<item>> sort="""$sort$""" itemClassFilter="""$itemClassFilter$""" exclude=<<excluded>> path=<<path>> field=<<field>>/>
+          <$list filter="""[is[current]get<tocP-tree-linker-field-name>]""" variable="newFields">
```

Comment:
`[all[current]` ... should be faster as `is[current` 

Is it possible to use [get<tocP-tree..>] .... Not sure if get uses currentTiddler by default.

Reply:
Thanks for the hint `is[current]` vs `all[current]`... nice insight! `get` uses the incoming tiddler-list.